spring/spring-boot/infinispan-spring-boot-starter/src/test/java/test/infinispan/autoconfigure/CacheDisabledTest.java

```diff
@@ -0,0 +1,46 @@
+package test.infinispan.autoconfigure;
```

Comment:
wrong package

Reply:
I'm not going to change this kind of things in this PR
I will open a PR after to make the tests better. there is already a JIRA on that

Reply:
so... what kind of changes are allowed in this PR?